lib/class-wp-rest-pattern-directory-controller.php

```diff
@@ -84,7 +84,7 @@ public function get_items_permissions_check( $request ) { // phpcs:ignore Variab
 	 * @return WP_Error|WP_REST_Response Response object on success, or WP_Error object on failure.
 	 */
 	public function get_items( $request ) {
-		$query_args  = array();
+		$query_args  = array( 'per_page' => 100 );
```

Comment:
Since Gutenberg is the primary consumer of the api.wordpress.org/patterns/ API, could this be made the default of the API instead of passing it?

Reply:
Removed - f57bafa6e26192a10e889b201e7dc537cdafc756

I haven't updated the wporg api yet.